tests/integration/main_test.go

```diff
@@ -60,9 +60,21 @@ func TestMain(m *testing.M) {
 func ConfigureTestEnv(testenv env.Environment) {
 	const receiverMockNamespace = "receiver-mock"
 
+	// List of the namespaces must match values/values_helm_opentelemetry_operator_enabled.yaml
+	// opentelemetry-operator.manager.env.WATCH_NAMESPACE value
+	var openTelemetryOperatorNamespaces = [...]string{"ot-operator1", "ot-operator2"}
+
+	// Helm chart installation extra arguments list
+	helmExtraArgs := []string{"--wait"}
```

Comment:
These are not really helm extra args but helm install extra args.

I believe we can actually put this into the `SetHelmOptionsTestOpt` and not expose here 🤔